scorecard.js

```diff
@@ -0,0 +1,39 @@
+class Scorecard {
+  constructor() {
+    this.score = 0
+    this.frame = 1
+    this.roll = 1
+  }
+
+  displayScore() {
+    return this.score
+  }
+
+  displayFrame() {
+    return this.frame
+  }
+
+  displayRoll() {
+    return this.roll
+  }
```

Comment:
I believe technically these are not needed. You can call 'instance_of_Scorecard'.score and it will return the score even without the method created. The same for other 2